setup.py

```diff
@@ -28,8 +28,9 @@
 
 INSTALL_REQUIRE = [
     "torch>=1.11",
-    "optimum==1.27.*",
-    "transformers>=4.36,<4.54",
+    # "optimum==1.27.*",
+    "optimum-onnx @ git+https://github.com/huggingface/optimum-onnx.git@transformers454",
+    "transformers>=4.45,<4.56",
```

Comment:
minimum transformers version now set to v4.45 which was released 1 year ago cc @IlyasMoutawwakil  @helena-intel @nikita-savelyevv